x/marker/spec/07_events.md

```diff
@@ -0,0 +1,173 @@
+
+Fires when a marker is finalized.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_finalized       | denom                 | {denom string}            |
+| marker_finalized       | administrator         | {admin account address}   |
+
+
+
+---
+## Activate
+
+Fires when a marker is activated.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_activated       | denom                 | {denom string}            |
+| marker_activated       | administrator         | {admin account address}   |
+
+
+---
+## Cancel
+
+Fired when a marker is cancelled successfully.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_cancelled       | denom                 | {denom string}            |
+| marker_cancelled       | administrator         | {admin account address}   |
+
+
+---
+## Destroy
+
+Fires when a marker is marked as destroyed and ready for removal.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_destroyed       | denom                 | {denom string}            |
+| marker_destroyed       | administrator         | {admin account address}   |
+
+---
+## Mint
+
+Fires when coins are minted for a marker.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_minted_coins    | denom                 | {denom string}            |
+| marker_minted_coins    | amount                | {supply amount}           |
+| marker_minted_coins    | administrator         | {admin account address}   |
+
+
+
+---
+## Burn
+
+Fires when coins are burned from a marker account.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_burned_coins    | denom                 | {denom string}            |
+| marker_burned_coins    | amount                | {supply amount}           |
+| marker_burned_coins    | administrator         | {admin account address}   |
+
+
+---
+
+Fires when coin is removed from a marker account and transferred to another.
+## Withdraw
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_withdraw_coins  | denom                 | {denom string}            |
+| marker_withdraw_coins  | amount                | {supply amount}           |
+| marker_withdraw_coins  | administrator         | {admin account address}   |
```

Comment:
would `transferred to address` be helpful in this Event?

Reply:
there are separate events from the bank indicating the transfers between addresses

Reply:
toAddress is part of this event, added to docs